internal/transactionsubmission/transaction_worker.go

```diff
@@ -186,7 +182,7 @@ func (tw *TransactionWorker) handleFailedTransaction(ctx context.Context, txJob
 				// Publishing a new event on the event producer
 				err = tw.produceSyncPaymentEvent(ctx, events.SyncErrorPaymentFromSubmitterType, &txJob.Transaction, store.TransactionStatusError)
 				if err != nil {
-					return fmt.Errorf("producing event transaction ID %s - Status %s: %w", txJob.Transaction.ID, txJob.Transaction.Status, err)
+					return fmt.Errorf("producing event Status %s: %w - Job %v", txJob.Transaction.Status, err, txJob)
```

Comment:
The error wrapping is incorrect here, it should end with the `: %w` error, instead of having it in the middle of the string.